test/build.js

```diff
@@ -0,0 +1,271 @@
+const test = require('ava');
+const brunch = require('../lib');
+const { prepareTestDir, teardownTestDir, fileContains, fileNotContains, fileExists } = require('./_test_helper');
+const fixturify = require('fixturify');
+
+test.beforeEach(() => {
+  teardownTestDir();
+  prepareTestDir();
+});
+
+test.serial.cb('basic build', t => {
+  fixturify.writeSync('.', {
+    'brunch-config.js': `module.exports = {
+      files: {
+        javascripts: {
+          joinTo: 'app.js'
+        }
+      }
+    };`,
+    'app': {
+      'assets': {
+        'index.html': '<h1>hello world</h1>'
+      },
+      'initialize.js': 'console.log("hello world")'
+    }
+  });
```

Comment:
Wow, this is actually looking pretty good. I have had an idea to create several brunch projects in the repo and operate on these, but this `fixturify` thing seems to be an even better option 👌